samples/nrf9160/aws_fota/prj.conf

```diff
@@ -0,0 +1,61 @@
+# General config
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_MPU_STACK_GUARD=y
+
+CONFIG_IMG_MANAGER=y
+CONFIG_BOOTLOADER_MCUBOOT=y
+
+# Enable DOWNLOAD libs
+CONFIG_NRF_DOWNLOAD_CLIENT=y
+CONFIG_NRF_DOWNLOAD_MAX_RESPONSE_SIZE=2048
+
+# Networking
+CONFIG_NETWORKING=y
+CONFIG_NET_SOCKETS_OFFLOAD=y
+CONFIG_NET_SOCKETS_SOCKOPT_TLS=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
+
+# LTE link control
+CONFIG_LTE_LINK_CONTROL=y
+CONFIG_LTE_AUTO_INIT_AND_CONNECT=n
+
+# BSD library
+CONFIG_BSD_LIBRARY=y
+
+# AT Host
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_AT_HOST_LIBRARY=y
+
+# MQTT
+CONFIG_MQTT_LIB=y
+CONFIG_MQTT_LIB_TLS=y
+
+# Allow writing to flash
+CONFIG_FLASH=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_MPU_ALLOW_FLASH_WRITE=y
+
+# Appliaction
+#CONFIG_MQTT_PUB_TOPIC="/my/publish/topic"
+#CONFIG_MQTT_SUB_TOPIC="/my/subscribe/topic"
+#CONFIG_MQTT_CLIENT_ID="my-client-id"
+#CONFIG_MQTT_BROKER_HOSTNAME="iot.eclipse.org"
+#CONFIG_MQTT_BROKER_PORT=1883
+
+# Main thread
+CONFIG_MAIN_THREAD_PRIORITY=7
+CONFIG_MAIN_STACK_SIZE=4096
+
+CONFIG_HEAP_MEM_POOL_SIZE=4096
+CONFIG_FLOAT=y
+CONFIG_MQTT_MAX_PACKET_LENGTH=2048
+#CONFIG_MBEDTLS_HEAP_SIZE=30000
+CONFIG_MBEDTLS_SSL_MAX_CONTENT_LEN=2048
```

Comment:
Do we use mbedTLS in this sample? I don't see it being enabled, unless it's done implicitly. Anyway, I think this file could use some cleanup, but I'll wait with more comments until the app is ready and we clearly see what do we need.

Reply:
You are correct mbedTLS is not used will remove it